2021-10-13-github-templates.md

```diff
@@ -0,0 +1,170 @@
+
+    ## Description
+    <!--- Describe your changes in detail -->
+    <!--- Please read the README before submitting pull requests for this project. -->
+
+    ## Motivation and Context
+    <!--- Why is this change required? What problem does it solve? -->
+    <!--- If it fixes an open issue, please link to the issue here. -->
+
+    ## How Has This Been Tested?
+    <!--- Please describe in detail how you tested your changes. -->
+    <!--- Include details of your testing environment, and the tests you ran to -->
+    <!--- see how your change affects other areas of the code, etc. -->
+
+    ## Screenshots (if appropriate):
+
+    ## Types of changes
+    <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+    - [ ] Bug fix (non-breaking change which fixes an issue)
+    - [ ] New feature (non-breaking change which adds functionality)
+    - [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+    ## Checklist:
+    <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+    <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+    - [ ] My code follows the code style of this project.
+    - [ ] My change requires a change to the documentation.
+    - [ ] I have updated the documentation accordingly.
+    - [ ] I have read the **CONTRIBUTING** document.
+    - [ ] I have added tests to cover my changes.
+    - [ ] All new and existing tests passed.
+    - [ ] I will abide by the code of conduct.
+```
+
+
+### Issues: Bug Reports
+
+```
+    <!--- Provide a general summary of the issue in the Title above -->
+
+    ## Before we start...
+
+    - [ ] I checked the documentation and found no answer
+    - [ ] I checked to make sure that this issue has not already been filed
+    - [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
+
+    ## Description
+    <!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+
+    ## Expected Behavior
+    <!--- Tell us what should happen -->
+
+    ## Actual Behavior
+    <!--- Tell us what happens instead -->
+
+    ## Possible Fix
+    <!--- Not obligatory, but suggest a fix or reason for the bug -->
+
+    ## Steps to Reproduce
+    <!--- Provide a link to a live example, or an unambiguous set of steps to -->
+    <!--- reproduce this bug. Include code to reproduce, if relevant -->
+    1.
+    2.
+    3.
+    4.
+
+    ## Context
+    <!--- How has this bug affected you? What were you trying to accomplish? -->
+
+    ## Your Environment
+    <!--- Include as many relevant details about the environment you experienced the bug in -->
+    * What branch/commit are you on?:
+    * Version used:
+    * Environment name and version (e.g. Chrome 39, node.js 5.4):
+    * Operating System and version (desktop or mobile):
+    * Link to your project:
```

Comment:
what does this one mean?

Reply:
I think it can be removed, it think if you were making use of for example dotenv_validator and then we can review the code from your github repo.